### PR TITLE
AC: Added coco_segm_precision metric

### PR DESCRIPTION
**Pull request by Anna-Mironova**

### CODE REVIEW

File: tools/accuracy_checker/accuracy_checker/metrics/__init__.py [hunk 1]
```diff
@@ -173,6 +179,7 @@
     'MSCOCORecall',
     'MSCOCOKeypointsPrecision',
     'MSCOCOKeypointsRecall',
+    'MSCOCOSegmAveragePrecision',
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/metrics/__init__.py:182]: is there plan to support `MSCOCOSegmRecall`?

**Review by eaidova — APPROVED** [review]

File: tools/accuracy_checker/accuracy_checker/metrics/coco_metrics.py [hunk 2]
```diff
@@ -256,6 +263,18 @@ def evaluate(self, annotations, predictions):
         return recalls
 
 
+class MSCOCOSegmAveragePrecision(MSCOCOAveragePrecision):
+    __provider__ = 'coco_segm_precision'
+
+    annotation_types = (CoCoInstanceSegmentationAnnotation, )
+    prediction_types = (CoCocInstanceSegmentationPrediction, )
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/metrics/coco_metrics.py:270]:
```suggestion
    prediction_types = (CoCoInstanceSegmentationPrediction, )
```

File: tools/accuracy_checker/accuracy_checker/metrics/coco_metrics.py [hunk 3]
```diff
@@ -21,12 +21,19 @@
     DetectionPrediction,
     DetectionAnnotation,
     PoseEstimationPrediction,
-    PoseEstimationAnnotation
+    PoseEstimationAnnotation,
+    CoCocInstanceSegmentationPrediction,
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/metrics/coco_metrics.py:25]:
```suggestion
    CoCoInstanceSegmentationPrediction,
```

Comment (Anna-Mironova) [reply tools/accuracy_checker/accuracy_checker/metrics/coco_metrics.py]: Fixed

File: tools/accuracy_checker/accuracy_checker/metrics/coco_metrics.py (hunk 2 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/accuracy_checker/metrics/coco_metrics.py]: Fixed

**Pull request merged**